packages/tab-scroller/src/mwc-tab-scroller-base.ts

```diff
@@ -32,18 +32,20 @@ export class TabScrollerBase extends BaseElement {
   @query('.mdc-tab-scroller__scroll-content')
   protected scrollContentElement!: HTMLElement;
 
+  /* eslint-disable @typescript-eslint/no-object-literal-type-assertion */
   @eventOptions({passive: true} as EventListenerOptions)
```

Comment:
I don't think we need this cast at all.

Reply:
Done! https://github.com/material-components/material-components-web-components/pull/371/commits/0fa2e5a717b4630ce1619162b9826dc054102c5a